content/swift/concepts/functions/functions.md

```diff
@@ -0,0 +1,172 @@
+---
+Title: 'Functions' 
+Description: 'Functions are self-contained chunks of code that perform a specific task.'
```

Comment:
```suggestion
Description: 'Functions are a type of structure that allows a set of operations to be packaged and repeated as desired.'
```

Reply:
I noticed that the original description was an exact copy of the language used in the swift documentation. That is considered plagiarism, and it is not permitted in docs. 

It appears that some of the examples, headers, and other content are also copied directly from the documentation. All content in docs must be original. I would advise that you update this entry to remove/replace all copied content.

Reply:
I don't mind to change the description but the header will always be the same because this are the things you need to talk about when talking about functions in swift, for examples I will change them you are right about that. @caupolicandiaz